### ISSUE TITLE
docker service create tab completion

### ISSUE THREAD

**Issue by lazize** [docker/docker#30910]
**Description**
Tab completion echo a strange message when use `docker service cre<TAB>`.

**Steps to reproduce the issue:**
1. Type `docker service cre<TAB>`

**Describe the results you received:**
When I press tab to make tab completion it echo the message "__docker_daemon_is_experimental: command not found", as you can see below.
```shell
$ docker service cre__docker_daemon_is_experimental: command not found
ate
```

**Describe the results you expected:**
When type: `docker service cre<TAB>`

I expect: `docker service create`

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**
```
$ docker version
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:50:14 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:50:14 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**
```
$ docker info 
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 12
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 85
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-62-generic
Operating System: Ubuntu 16.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.444 GiB
Name: vm1
ID: T27O:VWXY:CCB5:GUQX:D6DB:PSWF:BQWR:EP2N:LPOY:NWOP:SWDH:H3GC
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Http Proxy: http://138.106.75.10:3128/
Https Proxy: http://138.106.75.10:3128/
No Proxy: localhost,127.0.0.1
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VM running on VMWare Workstation 12

**Comment by albers** [comment]
Thanks very much for reporting. 
This issue was already reported in #30858, and there is a pending fix: #30875.

**Comment by AkihiroSuda** [comment]
Since this is tracked in the above issue and PR, I'm closing this issue.

**Comment by albers** [comment]
@lazize The issue will be fixed in a potential 1.13.2 release.
See https://github.com/docker/docker/pull/30875#issuecomment-279323995 for how to get a fixed bash completion script.

**Comment by lazize** [comment]
Thanks!
